### ISSUE TITLE
User removed from queue since version 1.31.0

### ISSUE THREAD

**Issue by PiotrPardo** [MyPureCloud/terraform-provider-genesyscloud#885]
Hello, 

After update to version 1.31.0 all users are removed from the queue:

```
  ~ resource "genesyscloud_routing_queue" "queue" {
        id                               = "queueID"
      ~ members                          = [
          - {
              - ring_num = 1
              - user_id  = "xxx"
            },
          - {
              - ring_num = 1
              - user_id  = "yyy"
            },
          - {
              - ring_num = 1
              - user_id  = "zzz"
            },
          - {
              - ring_num = 1
              - user_id  = "abc"
            },
          - {
              - ring_num = 1
              - user_id  = "def"
            },
        ]
        name                             = "Queue"
```

It's a big issue for us, because users are outside terraform config and now on each terraform run removed from queue.

**Comment by charliecon** [comment]
Hi @PiotrPardo 

First of all - thanks for reporting!

Just to clarify, you are:

1. Creating a routing queue using terraform.
2. Adding users to this queue via the Genesys Cloud UI (or some other way that isn't using terraform)
3. Then running `terraform apply` and overwriting the members of the queue?

I think I can see how the changes introduced in v1.31.0 would cause this behaviour. I would just like to recreate your scenario locally to confirm a few things.

-Charlie

**Comment by PiotrPardo** [comment]
Hi,

1. Queues are created via Terraform but members part is totally outside terraform, meaning member part is not included in schema.
2. Yes
3. Yes, in previous version there was no change. Since 1.31 all users are gone after apply.

**Comment by charliecon** [comment]
Understood. We will get an engineer working on this issue as soon as we can. 

Thanks again

(Tracking with DEVTOOLING-434)

**Comment by charliecon** [comment]
A fix for this problem has just been merged, and will be released on version 1.34.0 (scheduled for March 19th)
